core/node/eth_sender/src/eth_tx_aggregator.rs

```diff
@@ -402,6 +381,24 @@ impl EthTxAggregator {
             )
             .await
         {
+            if storage
+                .eth_sender_dal()
+                .get_number_of_failed_transactions()
+                .await
+                .unwrap()
+                != 0
```

Comment:
Does it make sense to move this check at the start of `loop_iteration`. Then we won't get info about `agg_op` but IMO it's useless in this case

Reply:
good idea